atomic-reactor.spec

```diff
@@ -75,9 +72,15 @@ Requires:       python3-PyYAML
 Requires:       python3-ruamel-yaml
 Requires:       python3-rpm
 Requires:       python3-six
+Requires:       koji
+Requires:       osbs >= 0.48
 Requires:       skopeo
-Provides:       python3-dock = %{version}-%{release}
-Obsoletes:      python3-dock < %{dock_obsolete_vr}
+Provides:       python3-atomic-reactor-koji = %{version}-%{release}
+Provides:       python3-atomic-reactor-metadata = %{version}-%{release}
+Provides:       python3-atomic-reactor-rebuilds = %{version}-%{release}
+Obsoletes:      python3-atomic-reactor-koji <= %{ar_subpackages_obsolete}
+Obsoletes:      python3-atomic-reactor-metadata <= %{ar_subpackages_obsolete}
+Obsoletes:      python3-atomic-reactor-rebuilds <= %{ar_subpackages_obsolete}
```

Comment:
nitpick: It would be nice to have the provides and obsoletes in pairs (see below) since that's what they are.

```
provides ...
obsoletes...
provides...
...
```

Reply:
sure